src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -151,154 +144,102 @@ export class SignInModal extends React.Component {
       <div className="container">
         <div className="row">
           <div className="columns small-12">
-            <h1>Sign in to VA.gov</h1>
-          </div>
-        </div>
-        <div className="row medium-screen:vads-u-display--none mobile-explanation">
-          <div className="columns small-12">
-            <h2>
-              One site. A lifetime of benefits and services at your fingertips.
-            </h2>
+            <h1 className="vads-u-margin-top--2 medium-screen:vads-u-margin-top--1 medium-screen:vads-u-margin-bottom--2">
```

Comment:
Will we run into any issues where the page content also defines an `H1`?

Reply:
^ No. Because of how the Modal accessibility is setup we should be safe.